apps/site/assets/ts/stop/components/StopPageRedesign.tsx

```diff
@@ -48,24 +49,33 @@ const StopPageRedesign = ({
     return <Loading />;
   }
 
+  const isBannerAlert = ({ effect }: Alert): boolean =>
```

Comment:
A bit of a misleading name since it technically has nothing to do with the banner.

Reply:
True will change to `isStopPageAlert` but open to other suggestions